go.mod

```diff
@@ -0,0 +1,36 @@
+module github.com/lytics/metafora
+
+go 1.12
+
+require (
+	github.com/coreos/bbolt v1.3.3 // indirect
+	github.com/coreos/etcd v3.3.15+incompatible
```

Comment:
Can we use the same version we use in `lio`?

Reply:
I've updated to 3.4.1, which is latest stable - Plan would be to do another upgrade in lio when we pull this (and grid) in at their new versions.